src/xlsx.rs

```diff
@@ -368,19 +368,38 @@ impl<RS: Read + Seek> Xlsx<RS> {
                             _ => (),
                         }
                     }
-                    if !state.is_very_hidden() {
```

Comment:
this change was different. we seem to completely skip very hidden sheets, where they include them but mark them as such